### DISCUSSION TITLE
saf-training/courses/user/06

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#6] [Ideas]
# saf-training/courses/user/06

The MITRE Security Automation Framework Team's Introduction to InSpec profile development

https://mitre.github.io/saf-training/courses/user/06.html

**Comment by drclphibbs** [comment]
In 6.1 the "Installation Tab" link gives 404 error.

**Reply by wdower** [reply]
Fixed. Thanks.

**Reply by grouse142** [reply]
What_Is_The_Target needs an example of a file path for a target (i.e., docker container in GitHub).

**Comment by karikarshivani** [comment]
In 6.1-2, we should probably mention that your local machine can also be a target

**Reply by karikarshivani** [reply]
6.2: ...trying to do in the first place. `inspec exec` tells InSpec to execute an existing profile.

**Reply by karikarshivani** [reply]
You could test against your local machine (which is less common and doesn't require you to use `-t`)

**Reply by karikarshivani** [reply]
We use the NGINX target that is running via docker in our Codespace environment by stating ` -t docker://nginx`

**Comment by karikarshivani** [comment]
Change `./lab-setup.sh` to `./build-lab.sh`

**Comment by karikarshivani** [comment]
Consider adding `--enhanced-outcomes` flag

**Comment by karikarshivani** [comment]
In the 'Want to give it a try?' section, maybe we can add a note about how all the metadata doesn't come through in the cli output but we will see its value in Heimdall

**Comment by p-oneil** [comment]
Moved to PR

**Comment by karikarshivani** [comment]
Consider moving the `--enhanced-outcomes` to reporter flag section (after cli)
